salt/modules/haproxyconn.py

```diff
@@ -13,16 +13,17 @@
 import os
 import logging
 
+log = logging.getLogger(__name__)
+
 try:
     import haproxy.cmds
     import haproxy.conn
     HAS_HAPROXY = True
 except ImportError:
+    log.error('haproxyctl not found.  Try pip install haproxyctl')
```

Comment:
Rather than cluttering up logs for people who are not going to use haproxy, this message should be returned by the `__virtual__` function like this:
```python
 def __virtual__():
    '''
    Only load the module if haproxyctl is installed
    '''
    if HAS_HAPROXY:
        return __virtualname__
    return (False, 'haproxyctl not found. Try pip install haproxyctl')
```